pkg/azurefile/azurefile.go

```diff
@@ -1031,3 +1121,56 @@ func (d *Driver) SetAzureCredentials(ctx context.Context, accountName, accountKe
 	}
 	return secretName, err
 }
+
+func (d *Driver) getAzcopyJob(dstFileshare string) (string, string, error) {
+	cmdStr := fmt.Sprintf("azcopy jobs list | grep %s -B 3", dstFileshare)
+	out, err := exec.Command("sh", "-c", cmdStr).CombinedOutput()
+	if err != nil && err.Error() != "exit status 1" {
+		return "", "", fmt.Errorf("couldn't list jobs in azcopy %v", err)
+	}
+	jobid, err := d.parseAzcopyJobList(string(out))
+	if err != nil {
+		return "", "", fmt.Errorf("parse jobid in azcopy get error %v", err)
+	}
+	if jobid == "" {
+		return jobid, "", nil
+	}
+	percent, err := d.parseAzcopyJobSummary(jobid)
+	return jobid, percent, err
+}
+
+func (d *Driver) parseAzcopyJobList(out string) (string, error) {
+	jobSegments := strings.Split(out, "JobId: ")
+	if len(jobSegments) < 2 {
+		return "", nil
+	}
+	jobSegments = jobSegments[1:]
+	for _, job := range jobSegments {
+		segments := strings.Split(job, "\n")
+		if len(segments) < 4 {
+			return "", fmt.Errorf("error parsing jobs list: %s", job)
+		}
+		statusSegments := strings.Split(segments[2], ": ")
+		if len(segments) < 2 {
+			return "", fmt.Errorf("error parsing jobs list status: %s", segments[2])
+		}
+		status := statusSegments[1]
+		if status == "InProgress" || status == "Completed" {
```

Comment:
@vibhansa-msft will such kind of azcopy be changed in the future? we are now relying on the azcopy output to resume azcopy job and monitor status.